HW-11/HW12/Homework_12.py

```diff
@@ -0,0 +1,78 @@
+import json
+from datetime import datetime
+
+# 1
+with open('phonebook.txt', 'r+', encoding='utf-8') as phone1:
+    try:
+        phone_book = json.load(phone1)
+    except Exception:
+        phone_book = []
+
+    while True:
```

Comment:
This is not a phonebook you implemented earlier. You made a lot of unnecessary changes. Please, use here previously approved phonebook implementation with only adding parts where you  work with a file
The whole point of this task is to keep your data in a file and basically you have only 3 cases when you need to access your file: the beginning of the program, adding a record and deleting a record. You made your solution according to this, but look at the 5th line of the code: you open your file and it keeps open during the whole running. I don't think your file should be opened for such big amount of time

Reply:
Code was changed according your comment